resource.address/http/src/main/java/org/kaazing/gateway/resource/address/http/HttpResourceAddress.java

```diff
@@ -426,6 +440,18 @@ private HttpServerHeaderOption() {
         }
     }
 
+    private static final class HttpUserAgentHeaderOption extends HttpResourceOption<Boolean> {
+        private HttpUserAgentHeaderOption() {
+            super(Kind.USER_AGENT_HEADER, "userAgentHeaderEnabled", Boolean.TRUE);
+        }
+    }
+
+    private static final class HttpHostHeaderOption extends HttpResourceOption<Boolean> {
+        private HttpHostHeaderOption() {
+            super(Kind.HOST_HEADER, "hostHeaderEnabled", Boolean.TRUE);
```

Comment:
Thus this should not be needed. `   A client MUST send a Host header field in all HTTP/1.1 request
   messages.` f from :https://tools.ietf.org/html/rfc7230#section-5.4